apps/builder/src/features/blocks/inputs/buttons/components/ButtonsItemSettings.tsx

```diff
@@ -53,6 +60,14 @@ export const ButtonsItemSettings = ({ item, onSettingsChange }: Props) => {
           onConditionChange={updateDisplayCondition}
         />
       </SwitchWithRelatedSettings>
+      <TextInput
+        label={t("blocks.inputs.button.buttonSettings.buttonValue.label")}
+        helperText={t(
+          "blocks.inputs.button.buttonSettings.buttonValue.helperText",
+        )}
```

Comment:
Usually we prefer to use `moreInfoTooltip` when we want to explain what the input is about. `helperText` is for when we need to provide indication on where to find the value for that input